android/java/org/chromium/chrome/browser/crypto_wallet/activities/AccountSelectorActivity.java

```diff
@@ -26,12 +27,16 @@
 import org.chromium.chrome.browser.crypto_wallet.util.WalletUtils;
 
 import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 
 public class AccountSelectorActivity extends BraveWalletBaseActivity
         implements AccountSelectorItemListener {
     private static final String TAG = "AccountSelector";
 
+    private static final List<Integer> SUPPORTED_COIN_TYPES =
+            Arrays.asList(CoinType.ETH, CoinType.SOL, CoinType.FIL);
+
```

Comment:
why no BTC?

Reply:
It's for DApps, and as far as I know we don't support Bitcoin (yet). I'm going to rename the constant into something more meaninful like `SUPPORTED_COIN_TYPES_ON_DAPP`.